scripts/etc/init.d/rcS

```diff
@@ -99,12 +99,37 @@ tee < ${LOGFILE}.pipe $LOGFILE &
 exec &> ${LOGFILE}.pipe
 rm ${LOGFILE}.pipe
 
+RPI1_HARDWARE="BCM2708"
+RPI2_HARDWARE="BCM2709"
+rpi_hardware=$(grep Hardware /proc/cpuinfo | cut -d " " -f 2)
+rpi1="false"
```

Comment:
could a single rpi_version variable do the same? empty => unknown, 1 => 1, 2 => 2...
version checking becomes easier, echoing the RPi version is easy, ...